### DISCUSSION TITLE
install frigate+ just to download new and better object

### DISCUSSION THREAD

**Discussion by kiloptero** [blakeblackshear/frigate#9337] [Ask A Question]
just want to donate something to the proyect and download new and better objects

any way to do that? dont care and upload my image

how can i do that? if i donate 50 bucks can i keep the new objects?
thanks!

**Comment by NickM-27** [comment]
What do you mean by download new and better object? It's not clear what you're asking.

**Comment by kiloptero** [comment]
ohh sorry, im talking if the objects detections with the frigate+ will be
better

Andres Roepke Del Solar.
Ingeniero Civil Industrial
Ingeniero Electronico
Mobile 88196932

**Reply by NickM-27** [reply]
I would suggest reading https://deploy-preview-6262--frigate-docs.netlify.app/plus/ it should answer your questions.

**Comment by kiloptero** [comment]
thanks, its not clear if only with the frigate+ will be better detection ..
thanks


Andres Roepke Del Solar.
Ingeniero Civil Industrial
Ingeniero Electronico
Mobile 88196932

**Reply by NickM-27** [reply]
Frigate+ has better detection due to it being trained on relevant images, there are other models out there that can be trained similarly and frigate can be configured to work with other models as well.
